src/lib/pickles_types/at_most.ml

```diff
@@ -106,7 +106,7 @@ module At_most_2 = struct
       type 'a t = ('a, Nat.N2.n) at_most
 
       include
-        Binable.Of_binable1
+        Binable.Of_binable1_without_uuid
```

Comment:
This change should be reflected in the binable version linter to assure stability of the first argument.